src/globus_cli/endpointish/endpointish.py

```diff
@@ -33,7 +33,7 @@ def __init__(
         log.debug("Endpointish.data=%s", self.data)
 
         log.debug("Endpointish determine ep type")
-        self.ep_type = EndpointType.determine_endpoint_type(self.data)
+        self.ep_type = EndpointType(self.data.get("entity_type"))
```

Comment:
Do we need to wrap this in error handling at all?
I want the CLI to be robust to potential new entity_type values being introduced.

The previous logic always made _a_ determination, even though it could be incorrect if new cases were introduced. I think we may want a catch-all type for the CLI when entity_type is unrecognized. Even if we don't carefully define the behavior for `EntityType.UNDEFINED` (good name?), it would help us be able to reason about what will happen when a new entity_type is introduced.

Reply:
Ah yeah, I'll make this able to handle a new entity_type.

Might not hurt to return `EntityType.UNDEFINED` if `entity_type` isn't present for some reason too?

Reply:
QQ: Should there be an "unrecognized" and an "undefined" entity type?

Reply:
If we're worried about the missing vs unrecognized ambiguity, I would go with `UNRECOGNIZED` and say that we don't yet recognize a missing `entity_type` as a meaningful/valid value. 😉